topics/iotsharp/index.md

```diff
@@ -0,0 +1,14 @@
+---
+aliases: iot-sharp
+created_by: maikebing
+display_name: IoTSharp
+github_url: https://github.com/IoTSharp
+logo: iotsharp.png
+related: iot 
+released: December 2018
+short_description: IoTSharp is an open source IoT platform .
+topic: iotsharp
+url: https://iotsharp.io/
+wikipedia_url: https://en.wikipedia.org/wiki/TikTok
```

Comment:
This is not the correct url, please fix.

Reply:
fixed